src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -233,6 +235,7 @@ public without sharing class UTIL_OrgTelemetry_SVC {
     private void handleNPSPSettings() {
         STG_SettingsService stgSvc = new STG_SettingsService();
         Addr_Verification_Settings__c addrVerifSettings = UTIL_CustomSettingsFacade.getAddressVerificationSettings();
+        Allocations_Settings__c allocationCustomSetting = UTIL_CustomSettingsFacade.getAllocationsSettings();
```

Comment:
I don't believe this is necessary. You can use `stgSvc.stgAllo` instead. See line 244.